.github/workflows/e2e.yml

```diff
@@ -28,6 +28,9 @@ jobs:
       - name: Install UI deps
         working-directory: ./ui
         run: npm ci
+      - name: Build the app and all packages
+        working-directory: ./ui
+        run: npm run build
```

Comment:
In theory we might also want to make this a dependency in the `turbo` config for running the e2e tests, but that gets kinda funky during local development because in that case you usually have already started in dev mode and have the code built and it'll get weird if you have competing builds. I'm inclined to say to leave it alone for now, BUT to add a note about it somewhere in the `e2e/README.md` getting started notes, so people know what to do if they hit issues with e2e in local dev.